_bioinfcomputing/access_methods.md

```diff
@@ -1,5 +1,209 @@
+
+Configuration is very quick and the process is described in this video:
+
+[![NoMachine](https://img.youtube.com/vi/I3znzb9M_Ms/0.jpg)](https://youtu.be/I3znzb9M_Ms "See how to connect with NoMachine")
+
+One of the nicest features of NoMachine is that you can disconnect from the server's desktop environment and later resume the connection and continue right where you left off with all terminal windows in the same place. NoMachine from Windows or Mac is often a better choice than installing a Linux desktop.
+
+### Screen and tmux
+GNU Screen and tmux are both terminal multiplexers. They run one or more shell processes on a remote system and enable you to disconnect and reconnect while leaving the process(es) running. They are both full-featured utilities that take a long time to learn. Using the command below, you can get working with tmux quickly.
+
+1. Use SSH to connect to a remote system: `ssh rhino`
+1. On Fred Hutch systems, load the tmux environment module: `module load tmux` or just `ml tmux`
+1. Start a tmux session by entering `tmux`, or attach to your running tmux session using `tmux attach` or just `tmux a`
+1. When in a session, typing `exit` or `ctrl-d` will exit the tmux session
+1. Closing the lid of your laptop and/or exiting your local ssh client will leave the tmux session running
+1. Explicitly disconnect with `ctrl-b d` (ctrl-b puts tmux into command mode, and the disconnect command is d)
+
+A note about tmux iterm2 integration. If you are using iterm2 on a Mac, add `-CC` to your tmux command. This will run a tmux console as well as tmux in that terminal. From there, you can use iterm2 commands to create a new tmux sessions in a new window (`command-n`) or a new tab (`command-t`). When you disconnect, simply reattach to that first tmux session, and iterm2 will re-open all your windows and/or tabs.
+
+A note about the X Window System and terminal multiplexers. As stated above, on your laptop you are running an X server, and the remote program is a client. Terminal multiplexing allow your laptop to disconnect because the multiplexing server is running on the remote system and your device is the client. When you close your laptop, the X server stops. Clients exit at that point as a client cannot do anything without a server. See [NoMachine](#nomachine-nx-(multi-os)) for a workaround for X programs.
+
+## Making Things Easier: The SSH config file
+Located in your home directory in the `.ssh` folder is a file called `config` (create it if it doesn't exist). Your SSH client will read configuration options from this file when you use it. Any of the command line options can be specified in this config file to avoid overly complex SSH commands and make frequent use of remote servers more straightforward.  
+
+### Forward X11
```

Comment:
What does "forward X11" mean?  What is this section about (for the newb)?

Reply:
I think this is about port forwarding with X11, which enhances the encryption security over just using an X window with SSH. But I am not certain on the details and I think someone from SciComp can give a better explanation of the motivation for doing this. I would also suggest changing the header to something like "X11 Forwarding" to make it more clear. Is @bmcgough the right person to follow-up on this?